lib/DDG/Publisher/Site/Duckduckgo/Root.pm

```diff
@@ -307,6 +307,13 @@ sub pages {{
 		js_page_type => 'Addons',
 		copyright_footer => 1,
 	},
+	app_info => sub {
+		%page_defaults,
+		no_hero_header_icon => 1,
+		js_page_type => 'Addons',
+		about_footer => 0,
+		copyright_footer => 0,
```

Comment:
Main difference here is the disabled footers. I'm re-using the `Addons` page type so the JS for the app page runs on this page as well.